### ISSUE TITLE
Is it possible to use JSONPath to pick the objects without a particular field?

### ISSUE THREAD

**Issue by denisvmedia** [ohler55/ojg#121]
I tried this program: https://go.dev/play/p/De9ncvFbK6S:

```go
package main

import (
	"fmt"

	"github.com/ohler55/ojg/jp"
	"github.com/ohler55/ojg/oj"
)

func main() {
	obj, err := oj.ParseString(`[{"name": "Alice", "age": 20}, {"age": 30}, {"name": "Bob"}]`)
	if err != nil {
		panic(err)
	}

	// Define the JSONPath expression to select objects without a "name" field.
	x, err := jp.ParseString("[?(!@.name)]")
	if err != nil {
		panic(err)
	}

	ys := x.Get(obj)
	fmt.Println(ys)
}
```

It won't panic, but it also just returns all the objects regardless the filter. Is it possible to achieve what I need with your library? Thank you.

**Comment by ohler55** [comment]
`[?(@.name exists false)]` should work but I tried it an it fails so it's a bug and I'll fix it tonight.

**Comment by ohler55** [comment]
Looks like I broke it in the last release when the `Nothing` value was introduced to be in close compliance with https://datatracker.ietf.org/doc/html/draft-ietf-jsonpath-base. Please try the develop branch for the fix. I'll release on confirmation.

**Comment by denisvmedia** [comment]
It works. Thank you.

**Comment by ohler55** [comment]
Released v1.18.1 with the fix. Thanks for letting me know about the issue.
